src/WireFormat.h

```diff
@@ -1605,6 +1648,182 @@ struct TakeIndexletOwnership {
     } __attribute__((packed));
 };
 
+struct TxParticipant {
+    uint64_t tableId;           // Table Id of the participant object.
+    uint64_t keyHash;           // Key Hash of the participant object.
+    uint64_t rpcId;             // Unique (per transaction) participant id.
```

Comment:
I'm not sure what "per transaction" means: can the same rpcId be used in different transactions (I would assume "no")?

This value is a unique RPC identifier, right?  Should we define a typedef for these to help clarify? If not, you'll need to be more clear in the documentation.